src/components/Code/example.js

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import { trim } from 'lodash';
+import styled from 'styled-components';
+
+import { Example, Info } from '../../utils/example';
+import { Input } from '..';
+import Code from '.';
+
+const Highlight = styled.span`
+  color: ${props => props.theme.colors.accent.blue};
```

Comment:
Please rebase this PR to capture the latest color naming changes.. Sorry for that :/